### PR TITLE
Sharing is caring

### PR DESCRIPTION
**Pull request by GeorgiaDrew**
### Blog title
Sharing is caring

### Blog description
Our journey from conference to 'makeshift' workshop 

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by ricktaylorHX** [comment]
taking :one:

**Comment by ricktaylorHX** [comment]
looks great! :+1:

**Comment by hxtonihurcombe** [comment]
Brilliant post guys and an interesting read. 
I think it's great you have normalised the feelings of anxiety surrounding public speaking, attending workshops and just generally being put outside your comfort zone as well as explaining the pressures you felt you faced of trying to absorb as much as you can knowing that you would be coming back and presenting to your colleagues. You both done a great job in the workshop and I think this is a nice way to sum up your experience as a whole.

File: _posts/2016-03-08-sharing-is-caring.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Simply showing up to this conference felt worthwhile.
+Even with zero participation, we would have still walked away with the reassurance that we are approaching our day to day work in well-practiced and proven ways. However, because we got stuck in, we also gained insight into new testing techniques and behaviours, as well as having the opportunity to chat with other Testers about what they do.
+
+Essentially, a conference is what you make it.
+
+## Co-creating and running the workshop
+So, we have pages and pages of notes, and we have a bunch of awesome ideas for activities. All we need to do is implement all of this into an awe-inspiring, well-structured workshop, and we’ll be set.
+
+When it came to actually creating the workshop, we found that we both approach something like this very differently. Trying to balance this project between two individuals, one overly-meticulous, and the other hell-bent on the finish line, was a learning curve.
```
Comment (mattsaunby) [inline _posts/2016-03-08-sharing-is-caring.md]: I'd suggest dropping the comma between 'overly-meticulous' & 'and the other'

File: _posts/2016-03-08-sharing-is-caring.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Sharing is caring
+categories: testing
+author: georgia_andy
+tags: testing, sharing, workshop, conference
```
Comment (hxmattobee) [inline _posts/2016-03-08-sharing-is-caring.md]: I incorrectly told you these tags should be comma separated. They should in fact be space separated. Sorry.

Comment (HXandrewhart) [inline _posts/2016-03-08-sharing-is-caring.md]: Thanks @hxmattobee, I'll get on that change

**Comment by sairturner** [comment]
I'll be reviewer 2.

This is a great read!

Just a small suggestion, I'd choose another word to superiors - maybe senior people/team members?

You might want to consider (either in this article or just as a discussion between you) how you think you might approach this same situation another time in the future. Would your nerves still be there, would you pre-imagine how the workshop might play out - would that be worthwhile or prohibitive?

**Comment by HXandrewhart** [comment]
Thanks for the feedback @sairturner - totally agree with your suggestion and have made the change.
Thanks for the +1 :)

**Pull request merged**